edgedb-client/src/builder.rs

```diff
@@ -1143,10 +1155,85 @@ impl Builder {
                 }
             }
         }
+        #[cfg(feature="unstable")]
```

Comment:
This doesn't have to be future=unstable. It's okay to calculate idle timeout anyways.